docs/install.md

```diff
@@ -434,14 +434,16 @@ Select the following tabs for the installation description of your target.
         Click **Next**.
 
     1. Tab **System**: As **Machine** you can select `q35`, but the older default will work as well. We recommend the para-virtualised **VirtIO SCSI** controller, which should be the default.  
-        Activate the check box `Qemu Agent`.  
+        _**Important:** Activate the check box `Qemu Agent`._  
```

Comment:
Since all VM images do now have `systemd-logind`, hence ACPI, enabled OOTB, this step is actually not needed anymore. I suggest to remove it, and leave Proxmox defaults in place. Shutdown/reboot etc from Proxmox GUI will work without it.

Reply:
Ok.